Automattic/wp-calypso
# Switch to our fork of react-virtualized.

This helps us avoid a direct dependency on polyfills which aren't needed, by making it possible to compile `react-virtualized` with the rest of Calypso. For more context, see https://github.com/Automattic/react-virtualized/pull/1.

It also changes the import syntax, since our version of the package should be tree-shakeable.

This change alone likely won't have a meaningful impact in bundle size, but once the other two libraries that are still making direct use of `babel-runtime` (`wpcom` and `wpcom-proxy-request`) are reworked too, we should be able to drop the `core-js@2` polyfills from our bundles, resulting in a significant saving in bytes.

#### Changes proposed in this Pull Request

* Switch to our (public) fork of `react-virtualized`
* Change import syntax to use named imports
* Add `@automattic/react-virtualized` to the list of packages to compile

#### Testing instructions

Ensure that features making use of `react-virtualized` (e.g. the infinite post scroller in Reader) continue to work normally.
